relay/pkg/block/pull.go

```diff
@@ -0,0 +1,48 @@
+package block
+
+import "github.com/keep-network/tbtc/relay/pkg/btc"
+
+func (f *Forwarder) pullHeaderFromBtcNetwork(height int64) (*btc.Header, error) {
```

Comment:
Let's call it `pullHeaderFromBtcChain` to be consistent with what we have on the push side.